qemu/tests/set_link.py

```diff
@@ -189,10 +189,10 @@ def set_link_test(linkid, link_up, change_queues, operstate_always_up=False):
                                     operstate will never change,
                                     need set it to True.
         """
-        if link_up:
-            expect_status = "up"
+        if params["os_type"] == "linux":
+            expect_status = "up" if link_up else "down"
         else:
-            expect_status = "down"
+            expect_status = "2" if link_up else "7"
```

Comment:
Better use `win_media_connected`  and `win_media_disconnected` instead of fix value.